x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form.tsx

```diff
@@ -118,66 +150,115 @@ export function SloEditForm({ slo }: Props) {
     }
   };
 
+  const handleChangeCheckbox = () => {
+    setIsCreateRuleCheckboxChecked(!isCreateRuleCheckboxChecked);
+  };
+
+  const handleCloseRuleFlyout = () => {
+    history.replace({ search: '' });
+    setIsAddRuleFlyoutOpen(false);
+  };
```

Comment:
If the customer closes the flyout without creating any rule, the customer will stay on the edit slo page right? 
Is it correct than one flow could become:
1. create slo
2. redirect to edit slo with flyout open
3. close flyout
4. stay on edit slo
5. needs to save or cancel to go back to the dashboard with the created slo

Step 5 is a little bit concerning for the experience IMO. I wonder if the flyout should not be opened from the dashboard page? The flow becomes Create SLO / Edit SLO -> redirect to dashboard -> open flyout for sloId. 
Or maybe simpler, when closing the flyout, we could redirect to the dashboard page? 


I'm thinking out loud, need to try this scenarios. But what do you think?

Reply:
@kdelemme 

I see what you mean, but I personally don't think it's that confusing. The user sees the URL change, so she knows the SLO has been created successfully. 

If a user cancels the Add Rule flyout, she can always navigate back to the list by using the breadcrumbs at the top.

If you feel strongly about it, I can change it to navigate back to the list open cancelling the flyout.

Reply:
@kdelemme I've now updated the PR so upon cancelling the flyout, it navigates back to the SLO List.